examples/tv-app/android/java/LevelManager.cpp

```diff
@@ -44,8 +45,9 @@ void emberAfLevelControlClusterInitCallback(EndpointId endpoint)
 void LevelManager::NewManager(jint endpoint, jobject manager)
 {
     ChipLogProgress(Zcl, "TV Android App: LevelManager::NewManager");
-    uint16_t ep = emberAfFindClusterServerEndpointIndex(static_cast<chip::EndpointId>(endpoint), app::Clusters::LevelControl::Id);
-    VerifyOrReturn(ep != kEmberInvalidEndpointIndex && ep < kLevelManagerTableSize,
+    uint16_t ep = emberAfGetClusterServerEndpointIndex(static_cast<chip::EndpointId>(endpoint), app::Clusters::LevelControl::Id,
```

Comment:
But don't we need to set kLevelManagerTableSize to include the dynamic endpoint count?  I guess without that this stuff will still work right for the static endpoints and just fail for the dynamic ones, but...

Reply:
Yeah, i thought about it but, since it is an implementation for the tv-app specifically and not a common server implementation I didn't change it.  But I can change it